src/System.Net.Requests/tests/HttpWebRequestHeaderTest.cs

```diff
@@ -91,5 +91,16 @@ public void Range_Add_Success(Uri remoteServer)
             }
         }
 
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_EndGetRequestStreamContext_Null(Uri remoteServer)
+        {
+            System.Net.TransportContext context;
+            HttpWebRequest httpWebRequest = HttpWebRequest.CreateHttp(remoteServer);
+            httpWebRequest.Method = "POST";
+            using (httpWebRequest.EndGetRequestStream(httpWebRequest.BeginGetRequestStream(null, null), out context))
+            {
+                Assert.Equal(null, context);
```

Comment:
Is this assert going to fail when this test runs on the full framework? (Also, if we keep the assert, it can use Assert.Null)